packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/recent-workspace.tsx

```diff
@@ -24,6 +25,8 @@ type TRecentWorkspace = {
 };
 
 const RecentWorkspace = observer(({ workspace, index }: TRecentWorkspace) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
```

Comment:
looks like you're not using it anywhere
or it's some kind of refactoring in parallel?

Reply:
i just wanted to remove isMobile from shared and use the is_mobile from ui store 🙏

Reply:
I see, great 💪